code/modules/mob/living/carbon/alien/examine.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/carbon/alien/examine_what_am_i(skip_gloves = FALSE, skip_suit_storage = FALSE, skip_jumpsuit = FALSE, skip_shoes = FALSE, skip_mask = FALSE, skip_ears = FALSE, skip_eyes = FALSE, skip_face = FALSE)
+	var/examine_color = "#473879"
+	return ", an <b><font color='[examine_color]'>alien</font></b>!"
```

Comment:
is there any reason why you don't just include the hex color code in the return statement?

Reply:
I might just rework this, it sucks